spec.html

```diff
@@ -28965,12 +28965,19 @@ <h1>
         <h1>
           HostEnsureCanCompileStrings (
             _calleeRealm_: a Realm Record,
+            _goal_: a grammar symbol,
```

Comment:
If only certain nonterminal will be passed, I would prefer to enumerate exactly those. Otherwise, I would prefer to take an enum.

Reply:
Changed to a list of possible nonterminals.